features/customers.feature

```diff
@@ -135,6 +135,25 @@ Scenario: Lock a customer
     And I press the "Search for Customer" button
     Then I should see "true" in the "Locked" field
 
+Scenario: Delete a customer with no id provided
+    When I visit the "Home Page"
+    And I press the "Delete" button
+    Then I should see the message "Server error!"
+
+Scenario: Delete a customer with id provided
+    When I visit the "Home Page"    
+    And I set the "First Name" to "John"
+    And I press the "Search for Customer" button
+    Then I should see "false" in the "Locked" field
```

Comment:
I think maybe this line could removed and we can make 148 start with And. Looks good otherwise